File: tests/fixtures/go/comments.go
// FIXTURE: Go file with various comment types
// TESTS: Minimal mode comment stripping

package main

// This is a standalone comment not adjacent to a declaration (STRIP)

/* This is a standalone block comment (STRIP) */

// Add adds two numbers together.
// This is a Go doc comment adjacent to a declaration (KEEP).
func Add(a int, b int) int {
	// This comment is inside a function body (KEEP)
	result := a + b // inline comment in body (KEEP)
	return result
}

// Regular comment not adjacent to declaration (STRIP)

// Calculator is a simple calculator.
// Multi-line Go doc comment (KEEP).
type Calculator struct {
	// Value field comment inside struct (STRIP - not a function body)
	value int
}

/* Block comment at module level not adjacent to declaration (STRIP) */

// NewCalculator creates a new Calculator.
func NewCalculator(value int) *Calculator {
	// body comment (KEEP)
	return &Calculator{value: value}
}

// Add adds to the calculator value.
func (c *Calculator) Add(x int) int {
	return c.value + x
}

// Standalone comment (STRIP)




// Test blank line normalization: 4+ blank lines above (normalize to 2)

var Version = "1.0.0"

File: tests/fixtures/ruby/comments.rb
# FIXTURE: Ruby file with various comment types
# TESTS: Minimal mode comment stripping

require 'json'

# This is a standalone comment (STRIP)

# Another standalone comment (STRIP)

# Add two numbers together.
# This is a RDoc-style doc comment adjacent to method (KEEP).
def add(a, b)
  # This comment is inside a method body (KEEP)
  result = a + b # inline comment in body (KEEP)
  result
end

# Regular comment between methods (STRIP)

# Calculator class documentation (KEEP).
class Calculator
  # Initialize the calculator (KEEP).
  def initialize(value)
    @value = value
  end

  # Add to stored value (KEEP).
  def add(x)
    # body comment (KEEP)
    @value + x
  end
end

# Standalone comment at module level (STRIP)




# Test blank line normalization: 4+ blank lines above (normalize to 2)

VERSION = "1.0.0"
